app/src/interfaces/_system/system-display-template/system-display-template.vue

```diff
@@ -76,20 +79,34 @@ const injectValue = computed(() => {
 
 	return { fields: [fakeVersionField] };
 });
+
+const tree = computed(() => {
+	if (props.fields) {
+		return { list: props.fields };
+	}
+
+	if (collection.value === null) {
+		return null;
+	}
+
+	const { treeList, loadFieldRelations } = useFieldTree(collection, injectValue);
+
+	return { list: treeList.value, pathLoader: loadFieldRelations };
+});
 </script>
 
 <template>
 	<div class="system-display-template">
-		<v-notice v-if="collection === null">
-			{{ t('interfaces.system-display-template.select_a_collection') }}
+		<v-notice v-if="tree === null">
+			{{ t('interfaces.system-display-template.select_a_collection_or_tree_fields') }}
```

Comment:
Is this message relevant to the end user? I think it should still be "Select a Collection" as this is what the user can control

Reply:
agreed. done.

Reply:
Agreed, although I believe that was done due to `useFieldTree` erroring if a collection is not provided.